RaveAddIn/XML/vbet.xml

```diff
@@ -28,13 +30,12 @@
       </Node>
       <Node label="Intermediates">
         <Children collapsed="true">
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_50']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_60']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_70']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_80']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_90']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_100']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='VBET_NETWORK']" type="vector" symbology="" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_50']" type="vector" symbology="threshold_50" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_60']" type="vector" symbology="threshold_60" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_70']" type="vector" symbology="threshold_70" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_80']" type="vector" symbology="threshold_80" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_90']" type="vector" symbology="threshold_90" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_100']" type="vector" symbology="threshold_100" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" />
```

Comment:
Thanks for adding in symbology keys here. Please:
- [ ] ditch the `xpathlabel` and use labels that mean something (remember that these are [thresholed and vectorized](https://riverscapes.github.io/vbet/algorithm.html#step-5---thershold-and-vectorize), but "unsantized". Maybe these nodes have a parent node of `Unsantized Raw Valley Bottom Extent` and then calli indviduals - `50% Threshold Raw VBET`
- [ ] I think you'll want to make each of these transparent if they are a solid color, but maybe you should just make them hollow (in which case they don't need transparent)

Reply:
Can do! @joewheaton